tests/expectations/tests/no-recursive-whitelisting.rs

```diff
@@ -25,6 +25,3 @@ fn bindgen_test_layout_Foo() {
 impl Clone for Foo {
     fn clone(&self) -> Self { *self }
 }
-impl Default for Foo {
-    fn default() -> Self { unsafe { ::std::mem::zeroed() } }
-}
```

Comment:
And this is once again falling out of our changing how blacklisted items are handled. (When we aren't recursively whitelisting, that is equivalent to treating all non-explicitly-whitelisted things as blacklisted).